numba/tests/test_unicode.py

```diff
@@ -221,6 +225,7 @@ def setUp(self):
 
 
 UNICODE_EXAMPLES = [
+    '',
```

Comment:
Perhaps add this locally here: https://github.com/numba/numba/pull/4715/files#diff-1033902199505251535a5e432f871c66R359 as the empty string is quite specific to this test?

Reply:
Looking at the rest of the tests it seems that it's often added as an extra e.g. https://github.com/numba/numba/blob/20690d2d8425c05cadf8d830d06395c28d6bba0d/numba/tests/test_unicode.py#L383-L384 (and many duplicates), perhaps it would be wise to add it here?

I think perhaps either add it here but de-duplicate it everywhere else, or just add it locally, up to you!

Reply:
Done.